notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListenerService.kt

```diff
@@ -2,19 +2,68 @@ package com.nathanrassi.notifique
 
 import android.service.notification.NotificationListenerService
 import android.service.notification.StatusBarNotification
+import android.util.Log
 import dagger.android.AndroidInjection
 import javax.inject.Inject
 
 class NotifiqueListenerService : NotificationListenerService() {
   @Inject internal lateinit var dao: Notifique.Dao
+  val TAG: String = "Notifique"
+  val notificationMap = mutableMapOf<String,MutableList<Int>>()
 
   override fun onCreate() {
     AndroidInjection.inject(this)
   }
 
   override fun onNotificationPosted(sbn: StatusBarNotification) {
+    val packageName: String = sbn.packageName.toString()
+    val notificationID: Int = sbn.id
+    if (uniqueNotification(packageName, notificationID)) {
+      //create notification object for db call
```

Comment:
Comments should be sentences.
https://publicobject.com/2014/06/28/comments-should-be-sentences/
https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html